File: Exercicio-09.c
#include <stdio.h>

int main(void) {

  int age, months, days, hours, minutes;

  printf("Digite a sua idade (Em anos): ");
  scanf("%d", &age);
  
  months = age * 12;
  days = age * 365;
  hours = age * 8760;
  minutes = age * 525600;
  
  printf("Meses: %d \n", months);
  printf("Dias: %d \n", days);
  printf("Horas: %d \n", hours);
  printf("Minutos: %d \n", minutes);
  
  return 0;
}

File: Exercicio-09.java
import java.util.Scanner;

public class Main
{
  public static void main (String[]args)
  {
    Scanner sc = new Scanner (System.in);
    int age, months, days, hours, minutes;

    System.out.printf("Digite a sua idade (Em anos): ");
    age = sc.nextInt();
  
    months = age * 12;
    days = age * 365;
    hours = age * 8760;
    minutes = age * 525600;
  
    System.out.println("Meses: " + months);
    System.out.println("Dias: " + days);
    System.out.println("Horas: " + hours);
    System.out.println("Minutos: " + minutes);
  }
}
